integration-tests/jail/disk/test.py

```diff
@@ -0,0 +1,333 @@
+#!/usr/bin/python3
+import json
+import requests
+import datetime
+import time
+
+server = 'http://localhost:9981'
+headers = {
+    'Content-Type': 'application/json',
+}
+
+
+def show(a):
+    print(json.dumps(a))
+
+
+def list_wallet():
+    q = {
+        "method": "wallet_list",
+        "jsonrpc": "2.0",
+        "params": [],
+        "id": "wallet_list"
+    }
+    data = json.dumps(q)
+    response = requests.post(server, headers=headers, data=data)
+    return response.json()["result"]
+
+
+def create_staking_address(name, passphrase):
```

Comment:
I think you can create a python package for rpc client.

Reply:
ok

Reply:
or use https://github.com/yihuang/crypto-chain-bot/blob/master/robot.py#L253 by @yihuang

Reply:
ok